src/ray/raylet/scheduling/cluster_task_manager.cc

```diff
@@ -144,11 +146,27 @@ void ClusterTaskManager::DispatchScheduledTasksToWorkers(
       auto &task = std::get<0>(work);
       auto &spec = task.GetTaskSpecification();
 
+      std::vector<std::unique_ptr<RayObject>> args;
+      bool success = true;
+      const auto &deps = spec.GetDependencyIds();
+      if (!deps.empty()) {
+        success = pin_task_arguments_(deps, &args);
+        if (!success) {
+          RAY_LOG(WARNING) << "Error getting task arguments from plasma store";
+        }
+        for (size_t i = 0; i < deps.size(); i++) {
+          if (args[i] == nullptr) {
+            RAY_LOG(INFO) << "Task " << spec.TaskId() << " argument " << deps[i] << " was evicted before task could be dispatched";
```

Comment:
Can you comment when this can happen? It happens only when some of objects are evicted before the whole bundle is pulled?